man/here.Rd

```diff
@@ -49,12 +49,15 @@ until a directory with at least one of the following conditions is found:
 \itemize{
 \item contains a file \code{.here}
 \item contains a file matching \verb{[.]Rproj$} with contents matching \verb{^Version: } in the first line
+\item contains a directory \code{.vscode}
+\item contains a file \verb{_quarto.yml}
+\item contains a file \code{renv.lock} with contents matching "\code{Packages}:\s*\{`
```

Comment:
Apparently more escaping is needed here. But it's not easily within the control of this PR 🤔 

```
* checking DESCRIPTION meta-information ...Warning: /home/runner/work/here/here/check/here.Rcheck/00_pkg_src/here/man/here.Rd:54: unknown macro '\s'
```